website/content/docs/configure-hcp-packer.mdx

```diff
@@ -0,0 +1,88 @@
+---
+description: |
+  Packer can be configured to publish artifact metadata for all completed builds to an active HCP Packer registry.
+  The configuration for HCP Packer is managed by environment variables for legacy JSON templates or via a hcp_packer_registry
+  block for HCL2 templates. 
+page_title: Configuring Packer for HCP Packer
+---
+
+# Configuring Packer for HCP Packer
+
+HCP Packer stores metadata about your Packer images so that you can track updates, use the most up-to-date base images, and deploy the most up-to-date downstream images. 
+For more details on How it Works see the [HashiCorp Cloud Platform Packer documentation](https://cloud.hashicorp.com/docs/packer#what-is-hcp-packer) 
+
+You can configure any Packer template to push metadata to the HCP Packer registry, but we recommend that you use HCL2 Packer templates. 
+The HCL2 support in Packer allows you to codify how Packer interacts with the registry and customize your image metadata. 
+
+Users wishing to use HCP Packer with JSON are encouraged to get started with the environment variables and eventually work to migrate from JSON to HCL2 configuration
+templates. 
+
+## HCP Packer Environment Variables
+
+HCP Packer configuration through environment variables is currently the only way to publish artifacts from a Packer JSON template. 
+
+The HCP Packer environment variables are made available for users who wish to publish their artifacts to an active HCP Packer registry
+without making any changes to their existing template. Making it an ideal solution for users looking to get started with an existing template; both JSON and HCL2.
+
+HCP Packer environment variables must be configured before executing a Packer build command. They work with all template types but are required for legacy JSON templates. 
+
+** Required Environment Variables**
+
+- `HCP_CLIENT_ID`         - The HCP client ID of the HashiCorp Cloud Platform service principle that will be authenticating Packer to the HCP Packer registry.
+
+- `HCP_CLIENT_SECRET`    - The HCP client secret of the HashiCorp Cloud Platform service principle that will be authenticating Packer to the HCP Packer registry.
+
+- `HCP_PACKER_BUCKET_NAME` -  The name of the Image bucket where Packer will store information about each artifact from a build within the configured template. 
+
+** Optional Environment Variables**
+
+- `HCP_PACKER_BUILD_FINGERPRINT` -  A unique identifier assigned to a build run to determine if metadata for a template build on the registry is complete. A fingerprint can 
+not be more than 32 characters long.
+
+- `HCP_PACKER_REGISTRY`    -  Allows users to toggle HCP Packer publishing off. Only valid when all required variables have been set. Allowed values are [0|OFF].
+
+~> **Build Fingerprint**: By default, Packer uses the HEAD Git SHA for the template file to create a fingerprint. 
```

Comment:
can we move this up to be under the build fingerprint? OR maybe we can incorporate this information into the description? I just want it to feel cohesive :)

Reply:
Good call ✅